Yoshi/Yoshi/YoshiConfigurationManager.swift

```diff
@@ -13,16 +13,36 @@ internal final class YoshiConfigurationManager {
     static let sharedInstance = YoshiConfigurationManager()
 
     private var yoshiMenuItems = [YoshiMenu]()
+    private var invocations: [YoshiInvocation]?
     private var presentingWindow: UIWindow?
     private weak var debugViewController: DebugViewController?
 
     /**
      Sets the debug options to use for presenting the debug menu.
 
      - parameter menuItems: The menu items for presentation.
+     - parameter invocations: The invocation types.
      */
-    func setupDebugMenuOptions(menuItems: [YoshiMenu]) {
+    func setupDebugMenuOptions(menuItems: [YoshiMenu], invocations: [YoshiInvocation]) {
         yoshiMenuItems = menuItems
+        self.invocations = invocations
+    }
+
+    /// Helper function to indicate if the given invocation should show Yoshi.
+    ///
+    /// - parameter invocation: Invocation method called.
+    ///
+    /// - returns: `true` if Yoshi should appear. `false` if not.
+    func shouldShow(invocation: YoshiInvocation) -> Bool {
+        guard let invocations = invocations else {
+            return false
+        }
+
+        if  invocations.contains(.all) {
+            return true
+        }
```

Comment:
I don't think we will need this if we define `.all` as a compound array case instead of its own case.